electrum/lnworker.py

```diff
@@ -1268,7 +1268,9 @@ async def pay_to_node(
                         trampoline_onion=trampoline_onion,
                         trampoline_fee_level=self.trampoline_fee_level,
                         trampoline_route=trampoline_route)
-                util.trigger_callback('invoice_status', self.wallet, payment_hash.hex())
+                util.trigger_callback('invoice_status', self.wallet, payment_hash.hex(), PR_INFLIGHT)
+                # s = self.wallet.get_invoice_status(self.wallet.get_invoice(payment_hash.hex()))
+                # self.logger.info(f"invoice status triggered (1) for key {payment_hash.hex()} and status {s}")
```

Comment:
self-note: it's much cleaner to trigger the `'invoice_status'` event in `set_invoice_status` when it actually changes (as-is already done). The reason it is *also* triggered here is to update the attempt-counter and progress for a lightning payment in the GUI.

maybe we should add a comment to that effect here, e.g. "to let GUI update payment progress (e.g. attempt counter)"